### DISCUSSION TITLE
Move other pipelines to Github Action

### DISCUSSION THREAD

**Discussion by chenrui333** [community/community#25065] [Actions]
<p>We have some pipelines running in Jenkins, some pipelines in Concourse, we want to consolidate into Github action land, I wonder if there are any good materials on this theme. Thanks!</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>There are no tools to migarate pipelines for other CI systems (such as Jenkins , Concourse you mentioned above) to Github actions directly. </p>
<p>Github actions workflow has its own syntax. You need to create them manually. For configuring a github workflow, please refer to this tutorial:</p>
<p><a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow</a> </p>
